### PR TITLE
PASSCOMP flag for mobs items - Pass through computers

### PR DESCRIPTION
**Pull request by JamieD1**
#### Changelog

Added PASSCOMP - Pass through computers for mobs/items

Feel free to suggest changes to items/mobs that should or should not have it.

:cl:  
rscadd: Added PASSCOMP - Pass through computers for mobs/items
/:cl:

### CODE REVIEW

**Comment by MacHac** [comment]
Wouldn't it make more sense to have a general PASSMACHINE that covers all full-tile machinery that blocks movement but not air?  Unless you're creating a very literal computer bug mob.

**Comment by JamieD1** [comment]
> Wouldn't it make more sense to have a general PASSMACHINE that covers all full-tile machinery that blocks movement but not air? Unless you're creating a very literal computer bug mob.

That was my original idea, however I cant remember now why I didnt do it, bear with me whilst I try and remember.

**Comment by Hopekz** [comment]
+1 to Mac's comment.

Very odd to make a passthrough flag for only 1 object can't really think of too many reasons you'd apply this in particular unless you do have that reason in mind that I'm missing.

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Having animals pass through things like computers and antimatter engines/smes feels too weird for me. Remove that please

**Comment by JamieD1** [comment]
> Having animals pass through things like computers and antimatter engines/smes feels too weird for me. Remove that please

One of the whole points was to let flying mobs go over computers, I got bullied by hopek to make it machines :(

**Comment by alexkar598** [comment]
Since when do cats fly?

**Comment by JamieD1** [comment]
> Since when do cats fly?

The whole cats on keyboard joke, get it? get it?

File: code/_globalvars/bitfields.dm [hunk 1]
```diff
@@ -98,7 +98,8 @@ GLOBAL_LIST_INIT(bitfields, list(
 		"PASSMOB" = PASSMOB,
 		"PASSCLOSEDTURF" = PASSCLOSEDTURF,
 		"LETPASSTHROW" = LETPASSTHROW,
-		"PASSDOOR" = PASSDOOR //yogs
+		"PASSDOOR" = PASSDOOR, //yogs
+		"PASSMACHINES" = PASSMACHINES
```
Comment (alexkar598) [inline code/_globalvars/bitfields.dm]: You forgot to add passcomputers to the list

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Review by alexkar598 — APPROVED** [review]

**Comment by alexkar598** [comment]
Merge cornflakes

**Pull request merged**